dev.py

```diff
@@ -558,19 +558,39 @@ def copy_openblas(cls, dirs):
         default `_distributor_init.py` file with the one
         we use for wheels uploaded to PyPI so that DLL gets loaded.
 
-        Assumes pkg-config is installed and aware of OpenBLAS.
+        Assumes pkg-config is installed and aware of OpenBLAS. 
+
+        The "dirs" parameter is typically a "Dirs" object with the
+        structure as the following, say, if dev.py is run from the
+        folder "repo":
+
+        dirs = Dirs(
+            root=WindowsPath('C:/.../repo'),
+            build=WindowsPath('C:/.../repo/build'),
+            installed=WindowsPath('C:/.../repo/build-install'),
+            site=WindowsPath('C:/.../repo/build-install/Lib/site-packages'
+            )
+
         """
         # Get OpenBLAS lib path from pkg-config
         cmd = ['pkg-config', '--variable', 'libdir', 'openblas']
         result = subprocess.run(cmd, capture_output=True, text=True)
+        # pkg-config does not return any meaningful error message if fails
         if result.returncode != 0:
-            print(result.stderrr)
```

Comment:
From this typo on `stderr`, I'm guessing that this tool is not used much, hence if any windows user can test it, that would be really nice.